contracts/colonyNetwork/ColonyNetwork.sol

```diff
@@ -111,6 +133,10 @@ contract ColonyNetwork is BasicMetaTransaction, ColonyNetworkStorage, Multicall
     colonyVersionResolver[_version] = _resolver;
     currentColonyVersion = _version;
 
+    if (!isMiningChain()){
+      skillCount = getChainId() << 128;
```

Comment:
Yeah, I ended up doing it this way so that, whenever we're dealing with a `skillId`, we know that `skillId` is unique and doesn't need any further modifications (consider the frontend, too, having to work out whether it needs to change anything about a `skillId` before querying e.g. the reputation endpoint).